extensions/amp-facebook-comments/1.0/test-e2e/test-amp-fit-text.js

```diff
@@ -0,0 +1,79 @@
+/**
```

Comment:
It looks like `amp-facebook-comments` doesn't have e2e tests in 0.1. If you want to introduce them for this feature feel free to do it in a later PR.

Reply:
Visual tests would probably be preferred to e2e for the full component however. Or unit tests for the scope of this PR.